Gaming The Syllabus/Assets/Scripts/Combat/CombatManager.cs

```diff
@@ -14,13 +14,16 @@ public enum GameState {
 
 public class CombatManager : Singleton<CombatManager>
 {
-    private static CombatUnit player;
+    public static GameState gameStatus;
+    public static CombatUnit player;
     private static CombatUnit[] enemies;
 
     private static List<CombatUnit> allCombatants;
 
     private static int indexOfCombatantWithCurrentTurn;
 
+    public static bool isDone = false; // Checks if this script is done running.
```

Comment:
This scares me, we should look into why this is necessary. We can adjust script execution order or think about our dependencies to make sure everything loads in the correct order